model/config.go

```diff
@@ -2194,17 +2194,23 @@ type PluginState struct {
 	Enable bool
 }
 
+type PublicKeyDescription struct {
+	Name string
+}
+
 type PluginSettings struct {
 	Enable                   *bool
 	EnableUploads            *bool   `restricted:"true"`
 	AllowInsecureDownloadUrl *bool   `restricted:"true"`
 	EnableHealthCheck        *bool   `restricted:"true"`
+	EnforceVerification      *bool   `restricted:"true"`
```

Comment:
3/5, but I'm wondering if we should force-on marketplace verification for now. Yes, it limits the ability for a customer to deploy a marketplace of plugins without signatures, but we can add that functionality later as we learn more. Adding this now commits us to supporting this going forward.

Reply:
User should be able to change `EnforceVerification` field and install unsigned plugins.

Reply:
So the intent behind the MVP is that we won't actually verify locally installed plugins at all: only plugins installed via the marketplace. So no need to change this field to be able to install unsigned plugins in the short term.

I'm proposing that we make marketplace plugin signature verification unconditional, so that we can think through plugin signing some more before we commit to a new configuration bit that might mean something else in the future. (`EnforceMarketplaceVerification`, `EnforceAllPluginVerification`, etc.)

Reply:
Let's do that. I'll remove `EnforceVerification` and verify every plugin in the `installPluginFromUrl` method.